v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/WriteToGCSUtility.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.transforms;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import java.util.Arrays;
+import org.apache.avro.Schema;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link WriteToGCSUtility} class provides the static values and methods to handle various File
+ * Formats.
+ */
+public class WriteToGCSUtility {
+
+    /* Logger for class. */
+    private static final Logger LOG = LoggerFactory.getLogger(WriteToGCSUtility.class);
```

Comment:
This will not be required once you make the other changes that I mentioned for this class.